src/loadConfigurationFile.js

```diff
@@ -60,5 +60,10 @@ module.exports = function(configPath) {
             throw new Error('Could not load required module loading for ' + chalk.underline(configPath));
         }
     }
-    return require(configPath);
+    var req = require(configPath);
+    if (typeof req === 'function') {
+        return req();
```

Comment:
@pago yes indeed. I just tried with the previous method of `parallel-weback -- --env.language=en` etc but it doesn't work.

If you could point me in the right direction I could take a look?